srtcore/core.cpp

```diff
@@ -6648,12 +6648,12 @@ void CUDT::processCtrl(CPacket& ctrlpkt)
       CGuard::enterCS(m_AckLock);
 
       // check the validation of the ack
-      int seqdiff = CSeqNo::seqcmp(ack, CSeqNo::incseq(m_iSndCurrSeqNo));
-      if (seqdiff> 0)
+      if (CSeqNo::seqcmp(ack, CSeqNo::incseq(m_iSndCurrSeqNo)) > 0)
```

Comment:
Small refactoring, logic is not changed.

Reply:
It's simply the value `seqdiff` ruled out. However, this variable was previously used in some further debug log expressions - these were also ruled out. Removal of this variable makes the use of seqcmp as comparison-only clear.